hw3/src/GDPChart.js

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import { PieChart, Pie, Tooltip, Cell, Legend } from 'recharts';
+import styles from './GDPChart.module.css';
+const GDPChart = () => {
+    const [data, setData] = useState([]);
+    const [error, setError] = useState(null);
+
+    const COLORS = ['#0088FE', '#00C49F', '#FFBB28', '#FF8042', '#FF6347', '#6A5ACD', '#32CD32', '#FFD700', '#DC143C', '#8A2BE2'];
```

Comment:
Some of the colors are really hard to read such as the yellow and orange since the page itself has a yellow/orange/red background

Reply:
i have updated the color contrast. but i had to compromise background color.